### PR TITLE
Split cf_quota resource into cf_org_quota and cf_space_quota

### LINKED ISSUE #69: Split cf_quota resource into cf_org_quota and cf_space_quota
The current [cf_quota](https://github.com/mevansam/terraform-provider-cf/blob/master/website/docs/r/quota.html.markdown) resources tries to provision two distinct underlying CF resource types, and have users select among the two types using the `org` attribute
* [Org quota definition /v2/quota_definitions](http://apidocs.cloudfoundry.org/1.24.0/organization_quota_definitions/creating_a_organization_quota_definition.html)
* [Space Quota Definition /v2/space_quota_definitions](http://apidocs.cloudfoundry.org/1.24.0/space_quota_definitions/creating_a_space_quota_definition.html)

> * `org` - (Optional) The Org within which this quota will be defined so it can be applied to spaces within that Org

The actual value of the org parameter is ignored, only its presence/absence seems used in the code:
https://github.com/mevansam/terraform-provider-cf/blob/0dfaf692eb587f5e94a1b44a129bde8f6bb7da67/cloudfoundry/cfapi/quota_manager..go#L85-L96

Since both the CF CLI and the CF CC API manage distinct objects (that may evolve independently), it would make more sense to expose two distinct CF resources. Also, trying to hide both concepts makes documentation wording complex and error prone.

> $ cf help -a | grep quota
>    quotas                                 List available usage quotas
>    quota                                  Show quota info
>    set-quota                              Assign a quota to an org
>    create-quota                           Define a new resource quota
>    delete-quota                           Delete a quota
>    update-quota                           Update an existing resource quota
>    space-quotas                           List available space resource quotas
>    space-quota                            Show space quota info
>    create-space-quota                     Define a new space resource quota
>    update-space-quota                     Update an existing space quota
>    delete-space-quota                     Delete a space quota definition and unassign the space quota from all spaces
>    set-space-quota                        Assign a space quota definition to a space
>    unset-space-quota                      Unassign a quota from a space
> 

Marking as 1.0 milestone to avoid backward incompatibilities

### PR DESCRIPTION
**Pull request by psycofdj**
This PR implements the split of `cf_quota` resource and data: fixes #69

### CODE REVIEW

File: website/docs/d/space_quota.html.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    org = "${cf_org.o1.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the space quota to look up
+* `org` - (Required) The organization within which the quota is defined
```
[the remark this answers is not in the document]
Comment (psycofdj) [reply website/docs/d/space_quota.html.markdown]: I don't see your point @gberche-orange. `org` guid is required to create a space quota. In a terraform use-case, if you want to use a pre-existing space quota, you need to import it using it's guid.

See [space quota](http://apidocs.cloudfoundry.org/1.27.0/space_quota_definitions/creating_a_space_quota_definition.html)

[the remark this answers is not in the document]
Comment (psycofdj) [reply website/docs/d/space_quota.html.markdown]: my bad, you're talking about `data` and I had in mind `resource`

**Comment by gberche-orange** [comment]
@psycofdj can you please resolve conflicts, so that when tests are green we can merge the pr ?

**Comment by psycofdj** [comment]
@gberche-orange I ran this test yesterday and it was OK on pcfdev. Let me run it again to be sure.

**Comment by janosbinder** [comment]
@psycofdj test is ok - only the p-mysql has issues. Please rebase/merge with dev, and afterwards it should be ok.

**Comment by psycofdj** [comment]
>  cloudfoundry_space_quota.10g-space: Server error, status code: 400, error code: 310001, message: Space Quota Definition is invalid: Total reserved ports must be -

Let me check this before merging

**Pull request merged**